specification/apicenter/resource-manager/Microsoft.ApiCenter/stable/2024-05-01/apicenter.json

```diff
@@ -3632,13 +3632,12 @@
         "properties": {
           "$ref": "#/definitions/ApiProperties",
           "description": "The resource-specific properties for this resource.",
-          "x-ms-client-flatten": true,
-          "x-ms-mutability": [
```

Comment:
Removing `x-ms-mutability` ensures the `update` command generated by aaz-dev includes parameters to define related payload properties. Without this change, the `update` command will have no parameters for the payload (e.g. --title, --type, --custom-properties, etc.)